neqo-transport/src/connection/mod.rs

```diff
@@ -2861,8 +2862,11 @@ impl Connection {
     {
         qdebug!([self], "Rx ACK space={}, ranges={:?}", space, ack_ranges);
 
+        let Some(path) = self.paths.primary_fallible() else {
```

Comment:
If we can eliminate .primary(), then we can use that name for this function...  Maybe we should do that now anyway.  We want to encourage people to use the fallible option anyway.

Reply:
I opened #1828 for this.